src/client/client.spec.js

```diff
@@ -51,4 +51,77 @@ describe("client test cases", () => {
       }
     });
   });
+
+  describe("Client Method Tests", () => {
```

Comment:
internal code doesn't have this describe block, could we mirror it?

Reply:
I'll publish a change to the internal code -- this is correct